dotnet/Samples/Translations/File/Batch/BatchFileTranslationService.cs

```diff
@@ -31,4 +38,31 @@ public static void Main()
 
         lwClient.TranslateBatchFile(translateBatchFileRequest);
     }
+
+    private static void TranslateBatchFileWithLinguisticOptions(ILanguageWeaverClient lwClient)
+    {
+        Dictionary<string, string> linguisticOptions = new Dictionary<string, string>();
+        linguisticOptions.Add("Spelling", "US");
```

Comment:
For having the same building pattern of requests, the linguistic options can be set like this:
`var translateBatchFileRequest = new TranslateBatchFileRequest()
{
    SourceLanguageId = "fra",
    TargetLanguageId = "eng",
    LinguisticOptions = new Dictionary<string, string>
    {
        {"Spelling", "US"}
     },
...
}`
Please update all the translation requests that use linguistic options.